base-api.Tests/V1/Helper/EntityHelper.cs

```diff
@@ -0,0 +1,20 @@
+using Bogus;
+using base_api.V1.Domain;
+
+namespace UnitTests.V1.Helper
+{
+    public class EntityHelper
+    {
+        public static ExampleEntity CreateExampleEntity()
```

Comment:
We could use AutoFixture to replace this helper method

Reply:
Good idea. I was also thinking about updating these tests to use FluentAssertions. Will create a separate ticket for these two test updates.

Reply:
Added [ticket](https://hackney.atlassian.net/browse/PA-56?atlOrigin=eyJpIjoiYWFiODAxNmMzNjNkNDQzZmFmMjA0Zjg4NGQ4MTA4NTMiLCJwIjoiaiJ9) to backlog pending team preferences.